chapter-06/recipe-03/cxx-example/generate.py

```diff
@@ -20,10 +20,11 @@
 
 primes = (number for number in numbers if is_prime[number])
 with open(output_file_name, 'w') as f:
-    f.write('#pragma once\n\n')
-    f.write('const size_t max_number = {0};\n'.format(max_number))
-    f.write('std::vector<size_t> & primes() {\n')
-    f.write('  static std::vector<size_t> primes;\n')
+    f.write('#pragma once\n')
+    f.write('#include <vector>\n\n')
+    f.write('const auto max_number = {0};\n'.format(max_number))
```

Comment:
I would keep using `size_t` here, as a matter of readabiity.

Reply:
OK